### PR TITLE
Added basic type definitions for react-relay

### PR DESCRIPTION
**Pull request by schickling**
1. Add a new type definition.
2. [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
3. [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
4. [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.
   ]

### CODE REVIEW

File: react-relay/react-relay.d.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+// Project: https://github.com/facebook/relay
+// Definitions by: Johannes Schickling <https://github.com/graphcool>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../react/react.d.ts"/>
+
+declare module "react-relay" {
+    import * as React from "react";
+
+    // fragments are a hash of functions
```
Comment (yuit) [inline react-relay/react-relay.d.ts]: Could you use JSDOC comment style?

**Comment by yuit** [comment]
@schickling thanks for the PR ! just small comment about using JSDOC style

File: react-relay/react-relay.d.ts (hunk 1 above)
Comment (schickling) [inline react-relay/react-relay.d.ts]: Could you quickly paste a fix here so I can apply it to the rest?

Comment (yuit) [inline react-relay/react-relay.d.ts]: /** Fragments are a hash of functions */

**Comment by schickling** [comment]
Please check again @yuit :)

**Comment by yuit** [comment]
@schickling  awesome 👍

**Pull request merged**